tests/ftests/077-sudo-cgreate_systemd_scope.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Test to create a systemd scope using cgcreate
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup, CgroupVersion
+from process import Process
+from libcgroup import Mode
+from run import RunError
+import consts
+import ftests
+import sys
+import os
+
+CONTROLLERS = ['cpu', 'pids']
+SLICE = 'libcgroup.slice'
+CGNAME = os.path.join(SLICE, '077cgcreate.scope')
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if Cgroup.get_cgroup_mode(config) != Mode.CGROUP_MODE_UNIFIED:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires the unified cgroup hierarchy'
+
+    return result, cause
+
+
+def setup(config):
+    return consts.TEST_PASSED, None
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    Cgroup.create(config, CONTROLLERS, CGNAME, create_scope=True)
+
+    # get the placeholder PID that libcgroup placed in the scope
+    pid = int(Cgroup.get(config, None, CGNAME, setting='cgroup.procs',
+                         print_headers=False, values_only=True, ignore_systemd=True))
+
+    return result, cause
+
+
+def teardown(config):
+    pid = int(Cgroup.get(config, None, CGNAME, setting='cgroup.procs',
+                         print_headers=False, values_only=True, ignore_systemd=True))
+    Process.kill(config, pid)
+
+    # systemd will automatically remove the cgroup once there are no more pids in
+    # the cgroup, so we don't need to delete CGNAME.  But let's try to remove the
+    # slice
+    try:
+        Cgroup.delete(config, CONTROLLERS, SLICE)
+    except RunError:
+        pass
+
+
+def main(config):
+    [result, cause] = prereqs(config)
+    if result != consts.TEST_PASSED:
+        return [result, cause]
+
+    setup(config)
+    if result != consts.TEST_PASSED:
```

Comment:
return from `setup()` is always `TEST_PASSED` and `result` here is return from `prereqs`, the return check can be removed.